### PR TITLE
Update the navbar logo url

### LINKED ISSUE #355: Navbar logo link is not working properly
**Describe the bug**
When a user clicks on the navbar logo, it should redirect to the home page. But currently, it reloads the same page.

**To Reproduce**
Steps to reproduce the behaviour:
1. Go to 'https://www.sack.edu.lk/contact_us' or any other page
2. Click on the logo on the navbar
3. See the error

**Expected behavior**
It should redirect to the homepage.

**Screenshots**
N/A

**Additional context**
N/A

**Are you working on it?**
No

### PR DESCRIPTION
**Pull request by Yoshitha-SACK**
## Purpose

The purpose of this PR is to fix #355

## Goals

Adding a correct link to the a href attribute in navbar.html

## Approach

I put the suitable link to the navbar.html/a href attribute.

### Screenshots

image

  
### Preview Link
https://yoshitha-sack.github.io/sack-site/

##  Checklist
- [x] This PR doesn't commit any keys, passwords, tokens, usernames, or other secrets.
- [ ] I have read and understood the development best practices guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by Yoshitha-SACK** [comment]
@Piumal1999

File: _includes/static/navbar.html [hunk 1]
```diff
@@ -4,10 +4,9 @@
     <div class="col">
       <div class="row justify-content-start">
         <div class="col-sm-4">
-          <a href="{{ site.baseurl }}">
+          <a href="../../">
```
Comment (Piumal1999) [inline _includes/static/navbar.html:7]:
```suggestion
          <a href="/">
```
I think we can use a single `/` for this. Try it and let us know

**Comment by Yoshitha-SACK** [comment]
Done now is it ok

File: _includes/static/navbar.html (hunk 1 above)
Comment (Yoshitha-SACK) [reply _includes/static/navbar.html]: Done

**Review by Piumal1999 — APPROVED** [review]
Good job @Yoshitha-SACK Thank you

**Pull request merged**